umbral/openssl.py

```diff
@@ -121,6 +121,39 @@ def _int_to_bn(py_int: int, curve: 'Curve'=None, set_consttime_flag=True):
         backend._lib.BN_set_flags(conv_bn, backend._lib.BN_FLG_CONSTTIME)
     return conv_bn
 
+@typing.no_type_check
+def _bytes_to_bn(bytes_seq: bytes, set_consttime_flag=True):
+    """
+    Converts the given byte sequence to an OpenSSL BIGNUM. 
+    If set_consttime_flag is set to True, OpenSSL will use constant time
+    operations when using this BIGNUM.
+    """
+    bn = _get_new_BN(set_consttime_flag)
+    backend._lib.BN_bin2bn(bytes_seq, len(bytes_seq), bn)
+    backend.openssl_assert(bn != backend._ffi.NULL)    
+    return bn
+
+@typing.no_type_check
+def _bn_to_bytes(bignum, length : int = None):
```

Comment:
Cool :+1:

Reply:
Hmm - unless I'm missing something this seems to break existing serialization conventions.  Why isn't this `Bignum.from_bytes` (perhaps it is invoked there)?

Reply:
Yep, this is invoked in `CurveBN.to_bytes()`